CMP1903_A1_2324/Testing.cs

```diff
@@ -1 +1,36 @@
-Empty
+using System;
+using System.Diagnostics;
+
+namespace CMP1903_A1_2324
+{
+    internal class Testing
+    {
+        /*
+         * This class should test the Game and the Die class.
+         * Create a Game object, call the methods and compare their output to expected output.
+         * Create a Die object and call its method.
+         * Use debug.assert() to make the comparisons and tests.
+         */
+
+        
+        //Method
+        public void Check()
+        {
+            Console.WriteLine("Testing die class...");
+            Die testDice = new Die();
+            testDice.RollDice();
+            Debug.Assert(testDice.Roll > 0 && testDice.Roll < 7);
+            Console.WriteLine("Rolls are confirmed to be within [1,6]");
+
+            Console.WriteLine("Testing game class...");
+            Game testGame = new Game();
+            for (int i = 0; i < 5; i++)
+            {
+                testGame.CreateDice();
+            }
+            testGame.CalcSum();
+            Debug.Assert(testGame.PrintSum() == testGame.DieList[0] + testGame.DieList[1] + testGame.DieList[2] + testGame.DieList[3] + testGame.DieList[4]);
+            Console.WriteLine("The output of the PrintSum() method of the Game class is equal to the sum of the first die roll + ... + last die roll");
+        }
```

Comment:
To include another debug.assert testing to aid in checking the sum is correct you could check the range between the minimum sum and maximum sum that could come from a three 6 sided dice because this would be beneficial in lower the risk of incorrect sums from the rolls.

To do this you would need to testing between the values of 3 (as the minimum) and 18(as the maximum) using the && operator to the check the values as follows >3 and <18.